src/main/kotlin/no/nav/dagpenger/inntekt/v1/InntektRoute.kt

```diff
@@ -43,6 +44,20 @@ fun Route.inntekt(inntektskomponentClient: InntektskomponentClient, inntektStore
             call.respond(HttpStatusCode.OK, storedInntekt)
         }
     }
+
+    route("inntekt/uklassifisert/update") {
+        post {
+            val request = call.receive<StoredInntekt>()
+
+            val compoundKey = inntektStore.getInntektCompoundKey(request.inntektId)
+
+            inntektStore.insertInntekt(
```

Comment:
We need to change `getInntektId` query in https://github.com/navikt/dp-inntekt-api/pull/16/files#diff-7e8bc0884244618cc4352f38312a789dR21 and get the latest inntektid instead?

Reply:
Ah, det hadde jeg glemt ja! Fikset nå